build/run-gulp-in-docker.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a script that runs gulp in a docker container,
+# for machines that doesn't have nodejs, go and java installed.
+
+# If the dashboard-build image doesn't exist, build it
+if [[ -z $(docker images | grep dashboard-build) ]]; then
+	docker build -t dashboard-build .
```

Comment:
Hmm... Can you tag the image with `<github repository url>-build`? dashboard-build seems generic enough that it may collide with other user's containers.